Examples/Algorithms/Vertexing/include/ActsExamples/Vertexing/SeedVertexFinderAlgorithm.hpp

```diff
@@ -0,0 +1,48 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/EventData/SimSpacePoint.hpp"
+#include "ActsExamples/Framework/DataHandle.hpp"
+#include "ActsExamples/Framework/IAlgorithm.hpp"
+
+#include <string>
+
+namespace ActsExamples {
```

Comment:
Same comment: I think we should make it explicit that it does single vertex finding.

Reply:
Renamed to `SingleSeedVertexFinderAlgorithm`.